js_client/src/interaction/inspectionPoints.ts

```diff
@@ -71,7 +74,8 @@ function createDebugEvent(): BeginDebugEvent {
         (a, b) => a.lineNumber - b.lineNumber
     );
 
-    const debugMessage = createDebugMessage(model.getLinesContent(), inspectionPoints).data;
+    const globalVariables: InspectionVariable[] = []
+    const debugMessage = createDebugMessageData(model.getLinesContent(), inspectionPoints, globalVariables);
```

Comment:
I dont understand why this is calling createDebugMessageData() and not createDebugMessage()